src/axes/numericAxis.ts

```diff
@@ -262,6 +278,29 @@ export module Axes {
       }
     }
 
+    /**
+     * Gets the approximate text width setting.
+     *
+     * @returns {boolean} The current text width approximation setting.
+     */
+    public usesTextWidthApproximation(): boolean;
+    /**
+     * Sets the approximate text width setting. Approximating text width
+     * measurements can drastically speed up plot rendering, but the plot may
+     * have extra white space that would be eliminated by exact measurements.
+     * Additionally, very abnormal fonts may not approximate reasonably.
+     *
+     * @param {boolean} The new text width approximation setting.
+     */
+    public usesTextWidthApproximation(enable: boolean): void;
+    public usesTextWidthApproximation(enable?: boolean): boolean {
+      if (enable == null) {
+        return this._usesTextWidthApproximation;
+      } else {
+        this._usesTextWidthApproximation = enable;
```

Comment:
Oh I'm incredibly foolish.

In the Plottable codebase. we try to stick with the fluent style, where getters return the gotten object and where setters return the calling object.  This allows for chaining.

In this scenario, after you set the variable here, please `return this;`.